test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval.test

```diff
@@ -439,7 +439,7 @@ Content-Type: application/xml
 </notifyContextRequest>
 =======================================
 NUMBER: 16
-TIMES: REGEX((0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 53, 65, 67, 69|0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 51, 63, 67, 69))
+TIMES: REGEX((0|1), (1|2|3), (3|4), (5|6), (9|10), (13|14), (14|15|16), (24|25|26), (34|35|36), (35|36|37|38), (37|38|39|40), (39|40|41|42), (51|52|53|54), (63|65|66), (67|72), (69|74))
```

Comment:
Yes of course, and the way you propose to do it is with the C preprocessor - CPP. The C preprocessor takes care of all '#-directives'.  #define, #include, #if¡, etc